mmv1/products/compute/HealthCheck.yaml

```diff
@@ -170,6 +170,23 @@ properties:
       The default value is 5 seconds.  It is invalid for timeoutSec to have
       greater value than checkIntervalSec.
     default_value: 5
+  - !ruby/object:Api::Type::Array
+    name: 'sourceRegions'
+    item_type: Api::Type::String
+    min_size: 3
+    max_size: 3
+    min_version: beta
+    description: |
```

Comment:
Could you please update the description to instead include information from [the REST API docs for this field](https://cloud.google.com/compute/docs/reference/rest/beta/healthChecks#HealthCheck.FIELDS.source_region)? This looks like gcloud documentation, which is confusing for users of the Terraform provider.

Reply:
Thanks for the review! Updated the description to match the REST API. Please take a look.